### DISCUSSION TITLE
3.1.0  update info

### DISCUSSION THREAD

**Discussion by wmdhosting** [wbrowar/craft-3-guide#47] [Ideas and Feature Requests]
New version is much better to work with..  :)   

1. Is it posiblle to add reorder of elements inside  box.. now it eaiser to drag and drop elemets from left list 
so Its better to have reorder . than moving elments from one box to other..
![Snag_6f979a94](https://user-images.githubusercontent.com/76176291/132241590-a9953dc9-88bd-4fc4-ad2f-bc0f53a59347.png)

2. Can you add  Snippet..  for Content stats..   like it was by default in content stats.. 
![Snag_6f9b62f6](https://user-images.githubusercontent.com/76176291/132241989-6da58e77-539e-4f1a-bfec-35febc0b40a1.png)

3 . Can you add template or snippet that show   Guides like accordion or Togglle. 
I tryed to show with css .. but do not work for some reason..

I think that much better  for end user and it takes less space..

**Comment by wmdhosting** [comment]
I found one mor thing.. 

Does nto save CSS selector..
![Snag_6fb0ef71](https://user-images.githubusercontent.com/76176291/132244215-bbef04ac-a34a-4774-9667-6ad039fe9809.png)

**Comment by wmdhosting** [comment]
Is it posiblle  to use  craft  slideouts..  for showing guides.. ?

**Comment by wbrowar** [comment]
Hi @wmdhosting, this is great feedback! I'm going to take some time to look into these and will keep you posted. I will definitely focus on the CSS selector first, since that seems like it must have broke in the last update.

Thanks for all the great screenshots and suggestions.
